### PR TITLE
Changes related to new logo

### PR DESCRIPTION
**Pull request by jasminejaksic**
Updated the logo with the new image, edited size, padding, etc and made other minor changes

### CODE REVIEW

**Comment by jasminejaksic** [comment]
Before and after screenshot
screen shot 2017-04-16 at 12 56 23 am

File: _includes/hero.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="hero">
   <div class="container">
-    <img class="logo" src="img/logo.jpg" alt="istio">
+    <img class="logo" src="{{ site.baseurl }}/img/logo.png" alt="istio">
```
Comment (geeknoid) [inline _includes/hero.html]: istio -> Istio

**Comment by geeknoid** [comment]
Note, we also need to replace all the imagery in the favicons directory. This is what gets displayed as icons in tabs and other similar places.

**Comment by rshriram** [comment]
@jasminejaksic I have uploaded favicons

**Comment by jasminejaksic** [comment]
@rshriram Should we use the blue version of the icon, since white is a bit difficult to see?

**Pull request merged**